spyder/utils/color_system.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+# Color scales used for the default dark palette
```

Comment:
```suggestion
"""
Extra colors used for the dark and light palettes in Spyder.
"""

```

I think it's better to use a docstring rather than comment here. What do you think about about my suggestion above?